_api-reference/profile.md

```diff
@@ -0,0 +1,754 @@
+[`profile.shards.searches.query`](#the-query-object) | Array of objects | Profiling information about the query execution.
+`profile.shards.searches.rewrite_time` | Integer | All Lucene queries are rewritten. A query and its children may be rewritten more than once, until the query stops changing. The rewriting process involves performing optimizations, such as removing redundant clauses or replacing query path with a more efficient one. After the rewriting process, the original query may change significantly. The `rewrite_time` field contains the cumulative total rewrite time for the query and all its children, in nanoseconds.
+[`profile.shards.searches.collector`](#the-collector-object) | Array of objects | Profiling information about the Lucene collectors that ran the search.
+[`profile.shards.aggregations`](#aggregations) | Array of objects | Profiling information about the aggregation execution.
+
+### The `query` object
+
+The `query` object contains the following fields.
+
+Field | Data type | Description
+:--- | :--- | :---
+`type` | String | The Lucene query type into which the search query was rewritten. Corresponds to the Lucene class name (which often has the same name in OpenSearch).
+`description` | String | Contains Lucene explanation of the query. Helps differentiate queries with the same type.
+`time_in_nanos` | Long | The time the query took to execute, in nanoseconds. In a parent query, the time is inclusive of execution times of all the child queries.
+[`breakdown`](#the-breakdown-object) | Object | Contains timing statistics about low-level Lucene execution.
+`children` | Array of objects | If a query has subqueries (children), this field contains information about the subqueries.
+
+### The `breakdown` object
+
+The `breakdown` object represents the timing statistics about low-level Lucene execution. Timings are listed in wall-clock nanoseconds and are not normalized. The `breakdown` timings are inclusive of all children times. The `breakdown` object is comprised of the following fields. All fields contain integer values.
+
+Field | Description
+:--- | :--- 
+`create_weight` | A `Query` object in Lucene is immutable. Yet, Lucene should be able to reuse `Query` objects in multiple IndexSearchers. Thus, `Query` objects need to keep temporary state and statistics associated with the index in which the query is executed. To achieve reuse, every `Query` object generates a Weight object, which keeps the temporary context (state) associated with the <IndexSearcher, Query> tuple. The `create_weight` field contains the time spent in the process of creating the Weight object.
```

Comment:
Should we replace "executed" with "run"?

Reply:
Since we're talking about low-level execution, I prefer "execute" in these cases. But I'll defer to @natebower to determine if it needs changing.